.changeset/seven-tips-burn.md

```diff
@@ -0,0 +1,7 @@
+---
+'@shopify/liquid-html-parser': minor
```

Comment:
I think this warrant a _separate_ changeset that is a `patch` change. Not really a new feature, so this is a patch.